Gemfile.tt

```diff
@@ -45,6 +46,7 @@ end
 
 group :test do
   gem "capybara"<%= gemfile_requirement("capybara") %>
+  gem "cucumber"
```

Comment:
Can you give me some more context on this change? We don't typically use cucumber because we almost never have a product owner who would actually look at the gherkin files so I have some doubts about it being in the default template

Reply:
@eoinkelly The reason I put it there to support our Readme.file generated in the newly generated app. So there it says

### Running the tests
bundle exec rspec spec
bundle exec cucumber
npm run lighthouse-spec

But when I run the cucumber it complains that there is no gem. 

Should we remove them both from read me and gem files?

Reply:
I removed the gem

Reply:
@emre-simtay I think we should also change the README (good catch btw)